### ISSUE TITLE
Remove remaning mentionings of tm:required

### ISSUE THREAD

**Issue by danielpeintner** [w3c/wot-thing-description#1680]
I think we have two places where `tm:required` is still used.

### A
 [Example 57](https://w3c.github.io/wot-thing-description/#td-model-example-relative-tmRef):

```
    "tm:required": [
        "/properties/innerTemperature",
        "/properties/outerTemperature"
    ]
```

### B
[12.1 application/td+json Media Type Registration](https://w3c.github.io/wot-thing-description/#media-type-section) 

Editor's note: Other Fragment Identifier Considerations

> There is in fact an assertion requiring them in TMs for tm:required.

**Comment by JKRhb** [comment]
This issue should be addressed by #1694.

**Comment by egekorkan** [comment]
Indeed, closing
